plugin/pkg/bastion/validator/admission.go

```diff
@@ -147,5 +148,13 @@ func (v *Bastion) Admit(ctx context.Context, a admission.Attributes, o admission
 	bastion.Spec.SeedName = shoot.Spec.SeedName
 	bastion.Spec.ProviderType = &shoot.Spec.Provider.Type
 
+	if bastion.Annotations == nil {
+		bastion.Annotations = map[string]string{}
+	}
+
+	if userInfo := a.GetUserInfo(); userInfo != nil {
+		bastion.Annotations[v1alpha1constants.GardenerCreatedBy] = userInfo.GetName()
+	}
+
```

Comment:
Nit: You could use `metav1.SetMetaDataAnnotation` :)

Reply:
Though, this annotation should only be set for the first creation and should not be changeable (this could go into the `pkg/apis/core/validation` package), or?